custom_components/saj_esolar/sensor.py

```diff
@@ -509,8 +513,9 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 
     """Setup the SAJ eSolar sensors."""
 
-    session = async_create_clientsession(hass)
-    data = SAJeSolarMeterData(session, config.get(CONF_USERNAME), config.get(CONF_PASSWORD), config.get(CONF_SENSORS), config.get(CONF_PLANT_ID))
+    session = async_create_clientsession(hass,verify_ssl=False) #some providers have broken SSL chains
```

Comment:
maybe the verify_ssl should be configurable? Im not sure it's a good idea to never check it, but adding it to the config is also annoying.

Reply:
Hass does not seem to trust the DigiCert Global Root G2 CA Certificate by default. Im not sure how else to solve the issue. Im open to suggestions.

Reply:
this might be the cause of the issue, seems they are migrating some of their clients certs quite  recently 
https://community.snowflake.com/s/article/check-impact-from-digicert-g2-certificate-update

Reply:
For now i will update the config with a extra option in the configuration.yaml to enable or disable SSL verification.